README.md

```diff
@@ -2,50 +2,184 @@
+and monitor performance with [Lighthouse CI](https://github.com/GoogleChrome/lighthouse-ci).
 
 **Features**:
 
 - ✅ Audit URLs using Lighthouse
-- 🎯 Test performance budget as a part of the build
+- 🎯 Test performance with LHCI assertions
+- 💵 Test performance with Lighthouse budgets
 - ⚙️ Full control over Lighthouse config
 - 🔍 Detailed output for quick debug
-- 🚀 Fast (less than 3 seconds) action initialization
+- 💾 Upload data to LHCI server
+- 🚀 Fast action initialization
 
 ## Usage
 
+### Basic Action
+
+> Use Case: Run Lighthouse on each push to the repo and save the results in action artifacts.
+
 Create `.github/workflows/main.yml` with the list of URLs to audit using lighthouse.
 The results will be stored as a build artifact.
 
+#### main.yml
+
 ```yml
-name: Lighthouse Audit
+name: Lighthouse
 on: push
 jobs:
   lighthouse:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v1
-      - name: Audit URLs using Lighthouse and ensure performance budget
+      - name: Audit URLs using Lighthouse
         uses: treosh/lighthouse-ci-action@v1
         with:
-          urls: |
-            https://example.com/
-            https://example.com/blog
-            https://example.com/pricing
-          budgetPath: ./budget.json
+          urls: 'https://example.com/'
+          runs: 1
+      # Note: Lighthouse-ci-actions overwrite .lighthouseci/ each run, therefore
+      # artifacts need to be saved after each run if using gh-actions artifacts.
       - name: Save results
-        uses: actions/upload-artifact@master
+        uses: actions/upload-artifact@v1
         with:
           name: lighthouse-results
-          path: './results'
+          # This will save the Lighthouse results as .json files
+          path: '.lighthouseci'
+```
+
+> Note: to view the reports follow the `temporary-public-storage` link printed
+> out in the action, or download the `json` files from the artifacts and open
+> them with the [Lighthouse Viewer App](https://googlechrome.github.io/lighthouse/viewer/)
+
+Link to `temporary-public-storage`:
+<img align="center" width="998" alt="Lighthouse CI Action" src="https://user-images.githubusercontent.com/6392995/68536792-76c06580-030d-11ea-8e19-c467e374434e.png">
+
+> Note: By default this action will also store the reports to LHCI
+> `temporary-public-storage` when a `lhci_server` is not specified, in order to
+> opt out, send the `disable_temporary_public_storage` parameter.
+
+[⚙️ See this workflow in use!](https://github.com/treosh/lighthouse-ci-action/actions?workflow=LHCI-multiple-urls)
+
+## Inputs
+
+### One of these Required
+
+#### `urls`
+
+Provide the list of URLs separated by a new line.
+Each URL is audited using the latest version of Lighthouse and Chrome preinstalled on user machine.
+
+```yml
+urls: |
+  https://example.com/
+  https://example.com/blog
+  https://example.com/pricing
+```
+
+#### `static_dist_dir`
```

Comment:
Let's move it to the optional section as staticDistDir or remove it.

Reply:
Should it? Using `staticDistDir` will override any `urls` sent and audit over the locally self hosted urls discovered in the `static_dist_dir`. `urls` and `staticDistDir` are mutually exclusive, so I think having it be required as an alternative to urls makes sense.

It is an _esoteric_ feature, but it's very useful for small static sites. I would like to keep it in as an option. And to me, if it was optional, then `urls` wouldn't really be required since if you used `staticDistDir` then you don't need `urls` and it'd be confusing to have a required-ish `urls` field when there is an optional field that makes it not-so-required.